### DISCUSSION TITLE
Feature Request: hide sensors from Web Server

### DISCUSSION THREAD

**Discussion by kknishev** [LibreHardwareMonitor/LibreHardwareMonitor#1239] [Ideas]
It would be great to be able to select sensors for the web server.
The web server is used with the HA Open Hardware Monitor integration and not all sensors are required for HA.

as an option, the ability to remove hidden sensors from the web server

**Comment by florentmorfaux** [comment]
Hijacking this thread a little bit..
I agree with the ability to choose sensors to be shown on the web monitor.
I would also add that showing the graphs there would be awesome too.
Finally I would love to see a section or another page to look at parsed historical data from logs.
(and fixing the missing favicon would help spotting the page quicker in bookmarks)
